umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md

```diff
@@ -27,17 +27,26 @@ In the menu, you can find a tab called _"Upgrade plan"_.
 * Clicking on the tab will direct you to the overview of the plans that you can upgrade to.
 * From here you can see the different plans, the price per month, and the limitations between each of the plans.
 
-![Upgrade plane step2](../images/upgrade\_plan.png)
+![Upgrade plane step2](../images/umbraco-cloud-plans-new.png)
 
 * If you are on a Starter plan you can upgrade to the Standard and the Professional plan.
 * If you are on the Standard plan you can upgrade to the Professional plan.
 
-To upgrade to the plan of your choosing, click on the green button for the plan that you want to upgrade to and then your project will be upgraded.
+Follow the below steps to upgrade your plan:
+1. Click on the button saying **"Select plan"** to choose the plan you want to upgrade to.
+2. In the next step, you can choose to upragde with to a **dedicated option**.
```

Comment:
```suggestion
2. Choose to upgrade to a **dedicated option** in the next step.
```

Reply:
Ah ok.. So maybe you can rephrase it to [Optional] Choose to upgrade to a **dedicated option** in the next step.
```suggestion
2. _[Optional]_ Choose to upgrade to a **dedicated option** in the next step.
```

Reply:
Yeah, I like that better 😊